src/main/scala/mesosphere/marathon/core/instance/Instance.scala

```diff
@@ -28,7 +28,8 @@ case class Instance(
     state: InstanceState,
     tasksMap: Map[Task.Id, Task],
     runSpec: RunSpec,
-    reservation: Option[Reservation]) extends Placed {
+    reservation: Option[Reservation],
+    role: String) extends Placed {
```

Comment:
I think we should use `type Role = String` as in #6961

Reply:
That's actually a pretty good idea, but then we might want to use that in the AppDefinition/PodDefinition as well, right? Where would be a good place to define the `type Role = String`?

Reply:
Done